versions.cfg

```diff
@@ -83,14 +86,9 @@ zope.keyreference = 4.2.0
 zope.mkzeoinstance = 4.1
 zope.password = 4.3.1
 
-# Newer versions than from the current Zope 4.7.  Remove after Zope updates it.
```

Comment:
Was this information not helpful? If a version of Plone is released before a version of Zope, how will you know which packages to pin?

Reply:
I will compare the Zope versions from 4.7 with 4.x and add those to our own versions.
I will likely do this only on the [5.2-dev](https://github.com/plone/buildout.coredev/blob/release/5.2-dev/versions.cfg) branch from which I make the releases.